From 98a39b8eee1c76ec3699f68310ff499f8ac348c6 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 8 Dec 2017 22:37:05 +0800
Subject: [PATCH 3091/5242] MLK-17128-2 ARM64: dts: imx8mq-evk: add max_snk_mw
 property for typec

commit  7a4b36c910554b8a730f01b031e9be3c581e28f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the max_snk_mw property to specify the max power of i.MX8MQ EVK
power limit is 18W, also change the max current to be 2A, which is
more matching the actual HW design.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 9a52180..fb26a18 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -447,8 +447,9 @@
 		src-pdos = <0x380190c8>;
 		snk-pdos = <0x380190c8 0x3802d0c8>;
 		max-snk-mv = <9000>;
-		max-snk-ma = <1000>;
+		max-snk-ma = <2000>;
 		op-snk-mw = <9000>;
+		max-snk-mw = <18000>;
 		port-type = "drp";
 		default-role = "sink";
 	};
-- 
1.7.9.5

